docs/pinot_hadoop.rst

```diff
@@ -48,151 +48,133 @@ Create a job properties configuration file, such as one below:
+   path.to.output=/user/pinot/output
 
-  # path.to.schema: Schema file for the table, stored locally
-  path.to.schema=flights-schema.json
+   # path.to.schema: Schema file for the table, stored locally
+   path.to.schema=flights-schema.json
 
-  # segment.table.name: Name of the table for which to generate segments
-  segment.table.name=flights
+   # segment.table.name: Name of the table for which to generate segments
+   segment.table.name=flights
 
-  # === Segment tar push job config ===
+   # === Segment tar push job config ===
 
-  # push.to.hosts: Comma separated list of controllers host names to which to push
-  push.to.hosts=controller_host_0,controller_host_1
-
-  # push.to.port: The port on which the controller runs
-  push.to.port=8888
+   # push.to.hosts: Comma separated list of controllers host names to which to push
+   push.to.hosts=controller_host_0,controller_host_1
 
+   # push.to.port: The port on which the controller runs
+   push.to.port=8888
 
 Executing the job
 ^^^^^^^^^^^^^^^^^
 
 The Pinot Hadoop module contains a job that you can incorporate into your
 workflow to generate Pinot segments.
 
-.. code-block:: none
+.. code-block:: bash
 
-  mvn clean install -DskipTests -Pbuild-shaded-jar
-  hadoop jar pinot-hadoop-<version>-SNAPSHOT-shaded.jar SegmentCreation job.properties
+   mvn clean install -DskipTests -Pbuild-shaded-jar
+   hadoop jar pinot-hadoop-<version>-SNAPSHOT-shaded.jar SegmentCreation job.properties
 
 You can then use the SegmentTarPush job to push segments via the controller REST API.
 
-.. code-block:: none
-
-  hadoop jar pinot-hadoop-<version>-SNAPSHOT-shaded.jar SegmentTarPush job.properties
+.. code-block:: bash
 
+   hadoop jar pinot-hadoop-<version>-SNAPSHOT-shaded.jar SegmentTarPush job.properties
 
 Creating Pinot segments outside of Hadoop
 -----------------------------------------
 
 Here is how you can create Pinot segments from standard formats like CSV/JSON.
 
 #. Follow the steps described in the section on :ref:`compiling-code-section` to build pinot. Locate ``pinot-admin.sh`` in ``pinot-tools/target/pinot-tools=pkg/bin/pinot-admin.sh``.
-#.  Create a top level directory containing all the CSV/JSON files that need to be converted into segments.
-#.  The file name extensions are expected to be the same as the format name (*i.e* ``.csv``, or ``.json``), and are case insensitive.
-    Note that the converter expects the ``.csv`` extension even if the data is delimited using tabs or spaces instead.
-#.  Prepare a schema file describing the schema of the input data. The schema needs to be in JSON format. See example later in this section.
-#.  Specifically for CSV format, an optional csv config file can be provided (also in JSON format). This is used to configure parameters like the delimiter/header for the CSV file etc.
-        A detailed description of this follows below.
+#. Create a top level directory containing all the CSV/JSON files that need to be converted into segments.
```

Comment:
We also support converting AVRO file to pinot segment. Can you mention it in this doc as well?

Reply:
Done